kinetic/distribution.yaml

```diff
@@ -5282,7 +5283,7 @@ repositories:
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/tork-a/jsk_3rdparty-release.git
-      version: 2.1.12-1
+      version: 2.1.12-2
```

Comment:
It seems strange (to me) to only bump the debinc, but also add a new sub-package.  Are there changes in the upstream source repository?  If so, I'd recommend a full release.